src/renderer/components/dock/dock.tsx

```diff
@@ -37,6 +37,8 @@ interface Dependencies {
   dockStore: DockStore
 }
 
+type Direction = 1 | -1;
```

Comment:
nit: would prefer if this was named:

```
enum Direction {
  RIGHT = 1,
  LEFT = -1,
```

Reply:
This was enum before, we decided to remove it.

Reply:
Oh strange okay...

I find enums more readable in code form, but okay.

Reply:
I moved to enum back and also simplifies the solution a bit